packages/mwp-tracking-plugin/src/util/trackingUtils.js

```diff
@@ -52,3 +57,10 @@ export const parseIdCookie = (
 	}
 	return parsed.id;
 };
+
+// ZoneId.toString() adds the timezone in brackets, which we want to remove
+// e.g.: 2019-01-07T10:27:02.791-05:00[America/New_York]
+export const getZonedDateTimeStringWithUTCOffset = () =>
```

Comment:
sort of an optimization, but this seems like a utility that should ideally be able to accept an arbitrary `ZonedDateTime` - what do you think about decomposing this into a function like

```js
const getISOString = zonedDateTime => zonedDateTime.toString().replace(/\[.*\]$/, '');
// 'Now' timezone defaults to NYC
const getISOStringNow = (zoneId = ZoneId.of('America/New_York')) =>
  getISOString(ZonedDateTime.now(zoneId))
```

I think those primitives might be easier to apply to more applications without making this primary usecase more awkward

Reply:
Yeah I think this is fair to separate them out. I'm not sure how often these will get used, but makes sense to do so.

Reply:
I did it slightly differently. Instead of accepting a zoneId, it takes the tz string. This way all of the js-joda ZoneId logic can be encapsulated within the utility methods. You won't need a potentially extra import in places you're using this method.